src/main/java/com/squareup/javapoet/CodeWriter.java

```diff
@@ -372,9 +385,9 @@ String lookupName(ClassName className) {
       return className.canonicalName;
     }
 
-    // If the class is in the same package, we're done.
+    // If the class is in the same package, we're done
```

Comment:
😮 

We might have to fight. I take it you don't subscribe to [comments should be sentences][1]?

 [1]: https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

Reply:
Forgive me master! 

I had initially appended to that comment because I had changed the conditional, but then backed it out. And I tried to restore the comment and forget the period. Reverted :)